kube/src/client/discovery.rs

```diff
@@ -108,9 +77,14 @@ impl ApiResourceExtras {
         ApiResourceExtras {
             scope,
             subresources,
-            operations,
+            operations: ar.verbs.clone(),
         }
     }
+
+    /// Checks that given verb is supported on this resource.
+    pub fn supports_operation(&self, operation: &str) -> bool {
+        self.operations.iter().any(|op| op == operation)
+    }
```

Comment:
Ah, this is actually very nice.

I thought we would have
````
#[derive(Serialize, Deserialize)]
#[serde(rename_all = "lowercase")]
pub enum Operations {
  Create,
  Get,
...
  Other(String),
}
```
but then your method avoids users needing to grab the verbs in scope if they wish (but they can still use the typed constants).